files/bootstrap.native/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "dnp_theme"
+github = "https://github.com/thednp/bootstrap.native"
+homepage = "http://themeforest.net/user/dnp_theme"
+description = "If you are looking for Bootstrap without jQuery or vanilla Javascript for Bootstrap, this is the thing."
+mainfile = "dist/bootstrap-native.min.js"
```

Comment:
Please tell me, should I also add files to root? or simply leave them in the dist folder?